### ISSUE TITLE
Index out of range when running cf app

### ISSUE THREAD

**Issue by Jonty** [cloudfoundry/cli#659]
````
	Aww shucks.

	Something completely unexpected happened. This is a bug in cf.
	Please file this bug : https://github.com/cloudfoundry/cli/issues
	Tell us that you ran this command:

		cf app spring-music

	using this version of the CLI:

		6.12.3-c0c9a03

	and that this error occurred:

		runtime error: index out of range

	and this stack trace:

		goroutine 1 [running]:
	main.generateBacktrace(0xc21000a000, 0x3)
		/var/vcap/data/gocd-agent/pipelines/Linux64-Unit-Tests/src/github.com/cloudfoundry/cli/main/main.go:117 +0xa9
	main.handlePanics(0x7f6215b79660, 0xc210197280)
		/var/vcap/data/gocd-agent/pipelines/Linux64-Unit-Tests/src/github.com/cloudfoundry/cli/main/main.go:101 +0x123
	runtime.panic(0x980a00, 0x129f337)
		/usr/local/go/src/pkg/runtime/panic.c:248 +0x106
	github.com/cloudfoundry/cli/cf/api.(*logNoaaRepository).GetContainerMetrics(0xc2102b6000, 0xc2102a5e40, 0x24, 0xc2102b6120, 0x1, ...)
		/var/vcap/data/gocd-agent/pipelines/Linux64-Unit-Tests/src/github.com/cloudfoundry/cli/tmp/cli_gopath/src/github.com/cloudfoundry/cli/cf/api/logs_noaa.go:68 +0x275
	github.com/cloudfoundry/cli/cf/commands/application.(*ShowApp).ShowApp(0xc21004e540, 0xc2102a5e40, 0x24, 0xc2101dca40, 0xc, ...)
		/var/vcap/data/gocd-agent/pipelines/Linux64-Unit-Tests/src/github.com/cloudfoundry/cli/tmp/cli_gopath/src/github.com/cloudfoundry/cli/cf/commands/application/app.go:169 +0x6e
````

**Comment by dkoper** [comment]
Hi Jonty!

Is the app deployed to a Diego cell or DEA? Can you reproduce the issue in the latest cf cli release?
https://github.com/cloudfoundry/cli/#downloads

We fixed an issue there related to stats in the API that is appearing in your trace.

Cheers,
* CLI Team

**Comment by Jonty** [comment]
Hi!

This was testing a new Diego deployment, however I can't easily reproduce the issue so I don't think it'll be possible to test the new client.

I'll assume the new cf cli fixes it and close this. Will let you know if it happens again after I upgrade.

**Comment by goehmen** [comment]
bug https://www.pivotaltracker.com/story/show/103051454 fixed in 6.12.4
